ununifi/assetlist.json

```diff
@@ -0,0 +1,34 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "ununifi",
+  "assets": [
+    {
+      "description": "The governance token of UnUniFi protocol.",
+      "denom_units": [
+        {
+          "denom": "uguu",
+          "exponent": 0
+        },
+        {
+          "denom": "guu",
+          "exponent": 6
+        }
+      ],
+      "base": "uguu",
+      "name": "UnUniFi",
+      "display": "guu",
+      "symbol": "GUU",
+      "logo_URIs": {
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/ununifi/images/ununifi.svg",
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/ununifi/images/ununifi.png"
+      },
+      "coingecko_id": "ununifi",
```

Comment:
Please remove this non-existent CoinGecko Id

Reply:
OK, delete this CoinGecko Id.